From: Bart Van Assche <bart.vanassche@wdc.com>
Date: Wed, 18 Jul 2018 08:58:02 -0700
Subject: RDMA/bnxt_re: Modify a fall-through annotation
Patch-mainline: v4.19-rc1
Git-commit: acd4307a21b5a9a7761aa55a5f5ca46cbd43f108
References: bsc#1104742 FATE#325917

This patch avoids that gcc reports the following warning when building
with W=1:

drivers/infiniband/hw/bnxt_re/ib_verbs.c:2404:4: warning: this statement may fall through [-Wimplicit-fallthrough=]

Signed-off-by: Bart Van Assche <bart.vanassche@wdc.com>
Signed-off-by: Jason Gunthorpe <jgg@mellanox.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/infiniband/hw/bnxt_re/ib_verbs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/infiniband/hw/bnxt_re/ib_verbs.c
+++ b/drivers/infiniband/hw/bnxt_re/ib_verbs.c
@@ -2408,7 +2408,7 @@ int bnxt_re_post_send(struct ib_qp *ib_q
 			default:
 				break;
 			}
-			/* Fall thru to build the wqe */
+			/* fall through */
 		case IB_WR_SEND_WITH_INV:
 			rc = bnxt_re_build_send_wqe(qp, wr, &wqe);
 			break;
